packages/components/src/components/input/input.component.ts

```diff
@@ -410,6 +438,8 @@ export default class SynInput extends SynergyElement implements SynergyFormContr
     const hasHelpText = this.helpText ? true : !!hasHelpTextSlot;
     const hasClearIcon = this.clearable && !this.disabled && !this.readonly;
     const isClearIconVisible = hasClearIcon && (typeof this.value === 'number' || this.value.length > 0);
+    const isDecrementStepperDisabled = this.type === 'number' && !this.noSpinButtons && (this.min ? this.valueAsNumber <= (typeof this.min === 'string' ? parseFloat(this.min) : this.min) : false);
```

Comment:
You`re right. I just checked the behavior of the standard html input[type=number]. If it is set to readonly, the stepper are no longer visible. Should we also just remove the stepper or just disable them?

Reply:
I think I would just disable them to not cause any visual clutter and it would also be inconsistent with the current components functionality. Maybe @mariohamann has another opinion on this?